### ISSUE TITLE
[Reload] FR: alternate method of reloading

### ISSUE THREAD

**Issue by lorensr** [meteor/mobile-packages#42]
I think a better user experience would be for the refresh to happen when the user leaves the app, like this:

```coffeescript
window.addEventListener 'blur', ->
  # reload if new version available
```

(Occasionally doesn't fire until app is brought back to foreground.)

Also, think it would be good to distinguish in the readme that you don't mean the user has to quit the app, just leave it (send to background) and return:

http://cordova.apache.org/docs/en/4.0.0/cordova_events_events.md.html#resume

**Comment by avital** [comment]
I think this is https://github.com/meteor/mobile-packages/tree/master/packages/mdg:reload-on-resume?

**Comment by avital** [comment]
Oh, your suggestion is to upgrade it on blur not only when you next run the app...

**Comment by stubailo** [comment]
Yeah, this is hard to do because we can't be sure that the code replacement will finish before the app is shut down for good. We'll look into it, though!

**Comment by lorensr** [comment]
Fixed by Meteor 1.3:

https://github.com/meteor/cordova-plugin-meteor-webapp
